app/controllers/account_controllers.rb

```diff
@@ -0,0 +1,46 @@
+
+get '/accounts/login' do
+  erb :login
+end
+
+post '/accounts/login' do
+  user = User.find_by(email: params[:email])
+  db_password = user.password_hash
+  if db_password == params[:password]
+    session[:current_user_id] = user.id
```

Comment:
modifying the session like this `session[:current_user_id] = user.id` should be done in an authentication helper method. Something like:

```ruby
helpers do
  def login_as(user)
    session[:current_user_id] = user.id
  end
end
```